processing/src/main/java/org/apache/druid/indexer/Checks.java

```diff
@@ -36,12 +37,18 @@ public static <T> Property<T> checkOneNotNullOrEmpty(List<Property<T>> propertie
         if (nonNullProperty == null) {
           nonNullProperty = property;
         } else {
-          throw new IAE("At most one of %s must be present", properties);
+          throw new IAE(
+              "At most one of %s must be present",
```

Comment:
Nit: brackets for interpolation

```suggestion
              "At most one of [%s] must be present",
```

Reply:
👍  Changed it to: `At most one of properties[%s] must be present` to also make it more obvious we're talking about properties.